### PR TITLE
Update content in the chapter "Test your service"

### PR DESCRIPTION
**Pull request by ydye**
introduce user to test their service with our new feature of single deploy and build

### CODE REVIEW

File: service-deployment/README.md [hunk 1]
```diff
@@ -120,15 +120,21 @@ If your service need more information, please add your property to cluster confi
 The new property should be placed in clusterinfo, machineinfo or machinelist.
 
 #### Test your service
-- ```sudo ./docker_build –p clusterconfig```
+
+- ```sudo ./prepare_hadoop_config.sh```
+
+Prepare hadoop configuration. If you don't sure whether your service depends on it or not. Please don't skip this step.
```
Comment (fanyangCS) [inline service-deployment/README.md]: >don't [](start = 37, length = 5)

"are not"

File: service-deployment/README.md [hunk 2]
```diff
@@ -120,15 +120,21 @@ If your service need more information, please add your property to cluster confi
 The new property should be placed in clusterinfo, machineinfo or machinelist.
 
 #### Test your service
```
Comment (fanyangCS) [inline service-deployment/README.md]: >Test [](start = 5, length = 4)

do you mean "(Re)Deploying on individual service"?

Comment (ydye) [reply service-deployment/README.md]: Yes, both test and redeploy individual service

File: service-deployment/README.md (hunk 1 above)
Comment (ydye) [reply service-deployment/README.md]: Resolved

**Pull request merged**